Generate javadoc failed for kafka-connect-adaptor

**Describe the bug**
When run `mvn deploy`, it throws the following exception
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:javadoc (default-cli) on project pulsar-io-kafka-connect-adaptor: An error has occurred in Javadoc report generation:
[ERROR] Exit code: 1 - javadoc: error - class file for com.google.protobuf.GeneratedMessageV3 not found
[ERROR]
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_291.jdk/Contents/Home/jre/../bin/javadoc @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in '/Users/hangc/Workspace/sourcecode/SN/pulsar.2.8.0/pulsar-io/kafka-connect-adaptor/target/site/apidocs' dir.
```

**To Reproduce**
1. Go to `pulsar-io/kafka-connect-adaptor`
2. run `mvn javadoc:javadoc`
3. See error

**Expected behavior**
generate javadoc success.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: mac
 - java: 1.8.0_291 (HotSpot)
 - maven: 3.6.2

**Additional context**
Add any other context about the problem here.

--- a/pulsar-io/kafka-connect-adaptor/pom.xml
+++ b/pulsar-io/kafka-connect-adaptor/pom.xml
@@ -113,5 +113,11 @@
       <type>test-jar</type>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
   </dependencies>
 </project>

